specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-10-01/network.json

```diff
@@ -0,0 +1,167 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2019-03-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
```

Comment:
Why is `text/json` necessary?

Reply:
Synchronized network.json with 2019-07-01 from network APIs.